README.md

```diff
@@ -0,0 +1,161 @@
+
+## How to submit
+You submit your result via a repo on GitHub.
+Clone this repo and point the remotes to your own public repo and you should be good to go.
+
+Make sure that you have a Git history with meaningful commits that tells the story of how you solved the task.
+A single init commit (or similar) contaning the entire solution is highly discouraged and will prevent us from doing proper evaluation.
+
+Make sure to provide guidance on how to run your solution in a `README.md`. Also use this file to highlight if there are any particular aspects of your solution you want us to consider.
+
+Make sure to fix the versions of any dependecies.
+This will ensure that the runtime environments will match.
+```
+# AVOID
+"some-package": "^0.3.1"
+
+# GOOD
+"some-package": "0.3.1"
+```
+Alternatively, you can create a Docker image holding the runtime environment.
+
+> ⚠️ **IMPORTANT:** A static Zip/RAR file containing the solution is not considered a valid submission and will be disregarded.
+
+Once you're ready to submit send link to the remote GitHub repo to **frontend-assignment@falcon.io**
+
+Unless agreed otherwise, we expect the submission to be handed in within 10 days. The net amount of time expected on the assignment is ~5-10 hours.
+If you, for any reason, estimate that you cannot finish the assignment within the timeframe please be communicative about an ETA.
+
+## How we evaluate
+When we evaluate your home assignemnt, before the onsite workshop, these are the main areas that we prioritize:
+
+#### Does your Git history reveal your workflow?
+When taking a first look at your Git repo we assess how you've iterated on the task.
+We do this by glancing through your Git commits. This will give us an idea of how you've priorised the work and how you've iterated on the development.
+E.g. how are business critical vs. non-critical implementations balanced?
+Therefore make sure that your commit messages provide specific context, e.g.:
+```
+# AVOID
+some fixes
+
+# GOOD
+fix(list): fix issues with the tasks list
+```
+
+#### Does your solution work out-of-the-box?
+Next, we read through your `README.md` and follow your instructions.
+Did you add custom steps? Do they work? Or, does it requires manual debugging to get up and running?
+
+#### To what degree are the requirements met?
+Once everything is up and running, dev server, API etc., we open your solution in a browser and click around.
+Does it work? Are the requirements met? If not, we ask "why" and seek for answers in `README.md` or code-level comments.
+
+#### How does it work?
+Provided that all of the acceptance criteria are up to standard, we start looking into how you implemented the solution.
+I.e. How did you decide to structure the application? How is it architectured? How do you manage state in the application? How scalable is your application?
```

Comment:
Since there are so many items here, each consideration could maybe be separated into its own list item. E.g.:

> - How did you decide to structure the application?
> - How is it architected?
> - ...

Also, _architected_ instead of _architectured_.

Reply:
_architectured_ is in fact the word we're looking for 😉